src/all/comicgrowl/src/eu/kanade/tachiyomi/extension/all/comicgrowl/PageResponse.kt

```diff
@@ -0,0 +1,16 @@
+package eu.kanade.tachiyomi.extension.all.comicgrowl
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class PageResponse(
+    val totalPages: Int,
+    val result: List<PageResponseResult>,
+)
+
+@Serializable
+data class PageResponseResult(
```

Comment:
Don't need data class.
```suggestion
class PageResponseResult(
```